src/App.scss

```diff
@@ -16,24 +20,60 @@ button {
 }
 
 .TodoInfo {
-  width: max-content;
+  width: 95%;
```

Comment:
Magic numbers?🥲

Reply:
I didn't spend much time on css and this one - helped me make my todo list look ok)